tests/session/session-startTransaction_error-006.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() throws an error on replicasets < 4.0
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto() ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '3.6'); ?>
+<?php skip_if_server_version('>=', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+$session = $manager->startSession();
+
+echo throws(function () use ($session) {
+    $session->startTransaction();
+}, MongoDB\Driver\Exception\RuntimeException::class), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
```

Comment:
If you're not using actual [EXPECTF](https://qa.php.net/expectf_details.php) patterns and merely doing an exact string match, you can use `EXPECT` here. Likewise in the second test below.